Components/Navigation/navigation.css

```diff
@@ -0,0 +1,59 @@
+/* Importing Common CSS File */
+
+@import "/common.css";
+
+/* Navbar Container */
+
+.nav-container {
+	position: sticky;
```

Comment:
Chaitanya, Loved the usage of sticky position instead of fixed. Learnt so many things in this article [sticky vs fixed](https://1stwebdesigner.com/sticky-vs-fixed-navigation-a-comparison/)